buildSrc/RustGenerator.js

```diff
@@ -183,10 +183,12 @@ function rustValueType(valueName, type, value) {
 	let innerType
 	if (valueName === "_id" && (type.type === Type.ListElement || type.type === Type.BlobElement)) {
 		if (value.type === ValueType.CustomId) {
-			innerType = "IdTupleCustom"
+			innerType = "Option<IdTupleCustom>"
```

Comment:
there's probably no way for us to avoid making IDs optional?

Reply:
or at least we would need a "here's data to create this thing" structure for each of them so yeah, just was wondering if you considered some options. I will admit (reluctantly) that this is probably the most pragmatic option